src/components/tiles/youtube-stream-tile.tsx

```diff
@@ -0,0 +1,19 @@
+import { Tile } from '@/components/tiles/tile';
+import { YoutubeVideo } from '@/components/youtube-video';
+
+type Props = {
+  title: string;
+  url: string;
+};
+export function StreamTile({ title, url }: Props) {
+  return (
+    <Tile className='sm:col-span-6 lg:col-span-3'>
+      <Tile.Body>
+        <h2 className='sm:text-2xl md:text-4xl'>{title}</h2>
```

Comment:
I think it would look better if this text was centered, and a bit of margin-bottom could help as well